core/src/trezor/ui/components/tt/confirm.py

```diff
@@ -38,8 +30,9 @@ def __init__(
         cancel_style: ButtonStyleType = DEFAULT_CANCEL_STYLE,
         major_confirm: bool = False,
     ) -> None:
-        super().__init__()
         self.content = content
+        button_confirm = None  # type: Optional[Button]
```

Comment:
please use the new-style type annotations: `button_confirm: Optional[Button] = None`

Reply:
Fixed in 81b9c568ba0a0f3e9e070a6d56990094d2937857. `# type: ignore` can't be expressed in the new style, right?

Reply:
correct; it's not a type, it's an instruction to the typechecker